catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -65,6 +65,16 @@ public void testSingleSchematron()
         getService("dog_legs.sch").validate(getMetacard("dog_4leg_3paw.xml"));
     }
 
+    @Test
+    public void testNullorEmptyMetadata()
+            throws ValidationException, IOException, SchematronInitializationException {
+        MetacardImpl card = new MetacardImpl();
+        SchematronValidationService service = getService("dog_legs.sch");
+        service.validate(card);
+        card.setMetadata("");
+        service.validate(card);
+    }
```

Comment:
How about something like `assertThat("No exceptions were thrown validating null/empty metadata", true, is(true))`? I don't like implicit assertions now that I think about it.